### PR TITLE
Add profile benjaminmichaelis.json

### PR DESCRIPTION
**Pull request by BenjaminMichaelis**
Adding Benjamin Michaelis profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1162"></a>

### CODE REVIEW

File: public/data/benjaminmichaelis.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "url": "https://twitter.com/benmichaelis",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Coding",
+      "date": "January 2020",
+      "icon": "code",
+      "color": "#gray",
```
Comment (kumarsonsoff3) [inline public/data/benjaminmichaelis.json:33]: Please use the correct **_Hex code_** for color!

```suggestion
      "color": "#787777",
```

File: public/data/benjaminmichaelis.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Benjamin Michaelis",
+  "type": "personal",
+  "bio": "Full time student, part-time adventure seeker/traveler",
```
Comment (kumarsonsoff3) [inline public/data/benjaminmichaelis.json:4]: It seems more attractive

```suggestion
  "bio": "Full time Student | Part-time adventure seeker | Traveler",
```

**Review by kumarsonsoff3 — APPROVED** [review]
You are requested to perform these changes!

File: public/data/benjaminmichaelis.json (hunk 2 above)
Comment (kumarsonsoff3) [inline public/data/benjaminmichaelis.json:4]: It seems more attractive

```suggestion
  "bio": "Full time Student | Part-time Adventure seeker | Traveler",
```

File: public/data/benjaminmichaelis.json (hunk 1 above)
Comment (kumarsonsoff3) [inline public/data/benjaminmichaelis.json:33]: Please use the correct **_Hex code_** for color!

```suggestion
      "color": "#787777",
```

File: public/data/benjaminmichaelis.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/benmichaelis",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Coding",
+      "date": "January 2020",
+      "icon": "code",
```
Comment (kumarsonsoff3) [inline public/data/benjaminmichaelis.json:32]: There is no icon for `"code"`.
**you can use `"Codewars"`, or `"Codeforces"` instead!**

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
You are Requested to perform these Changes

**Review by kumarsonsoff3 — APPROVED** [review]
**Nicely Done! 😀**

**Review by ssensalo — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**